src/regions.jrl

```diff
@@ -1102,3 +1102,32 @@ p({class:"foam.nanos.auth.Region",code:"PK-PB","isoCode":"PB",name:"Punjab",coun
 p({class:"foam.nanos.auth.Region",code:"PK-SD","isoCode":"SD",name:"Sindh",countryId:"PK"})
 p({class:"foam.nanos.auth.Region",code:"PK-JK","isoCode":"JK",name:"Azad Jammu and Kashmir",countryId:"PK","alternativeNames":["Āzād Jammūñ o Kashmīr"]})
 p({class:"foam.nanos.auth.Region",code:"PK-GB","isoCode":"GB",name:"Gilgit-Baltistan",countryId:"PK","alternativeNames":["Gilgit-Baltistān"]})
+
+//Ukrain
+p({class:"foam.nanos.auth.Region",code:"UA-71","isoCode":"71",name:"Cherkaska",countryId:"UA",alternativeNames:["Cherkashchyna"]})
+p({class:"foam.nanos.auth.Region",code:"UA-74","isoCode":"74",name:"Chernihivska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-77","isoCode":"77",name:"Chernivetska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-12","isoCode":"12",name:"Dnipropetrovska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-14","isoCode":"14",name:"Donetska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-26","isoCode":"26",name:"Ivano-Frankivska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-63","isoCode":"63",name:"Kharkivska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-65","isoCode":"65",name:"Khersonska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-68","isoCode":"68",name:"Khmelnytska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-35","isoCode":"35",name:"Kirovohradska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-32","isoCode":"32",name:"Kyivska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-09","isoCode":"09",name:"Luhanska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-46","isoCode":"46",name:"Lvivska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-48","isoCode":"48",name:"Mykolaivska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-51","isoCode":"51",name:"Odeska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-53","isoCode":"53",name:"Poltavska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-56","isoCode":"56",name:"Rivnenska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-59","isoCode":"59",name:"Sumska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-61","isoCode":"61",name:"Ternopilska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-05","isoCode":"05",name:"Vinnytska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-07","isoCode":"07",name:"Volynska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-21","isoCode":"21",name:"Zakarpatska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-23","isoCode":"23",name:"Zaporizka",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-18","isoCode":"18",name:"Zhytomyrska",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-43","isoCode":"43",name:"Avtonomna Respublika Krym",countryId:"UA"})
+p({class:"foam.nanos.auth.Region",code:"UA-30","isoCode":"30",name:"Kyiv",countryId:"UA"})
```

Comment:
@Mykola97 please let me know if this is valid - as it's a city rather than a province. Also with the Avtonomna above.

Reply:
It looks good according to the wikipedia there are 2 cities in the list. Kiev city and Kyivska province are usually separate even though one is part of another. Avtonomna Respublika Krym, Luhanska, Donetska is based on what you believe, since these parts of Ukraine are occupied by russia. Also, Avtonomna Respublika Krym was an independent part of Ukraine inside Ukraine, so it is not considered a province.